proto/gnmi_ext/gnmi_ext.proto

```diff
@@ -89,3 +92,47 @@ message TimeRange {
   int64 start = 1; // Nanoseconds since the epoch
   int64 end = 2;   // Nanoseconds since the epoch
 }
+
+// The CommitID is returned by the gNMI server to indicate the ID of a commit.
+// It is returned in response to the Set Request with the CommitConfirm message
+// that had COMMIT_CONFIRMED_ACTION_START set.
+// The CommitID is used in the subsequent SetRequest/CommitConfirm messages
+// to identify commit to be confirmed, reset or rejected.
+message CommitID {
+  string id = 1;
+}
+
+// The CommitConfirmed allows automatic rollback of a commit
+// if Set changes are not confirmed within the specified timeout.
+// The document about gNMI commit confirmed can be found at
+// https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-commit-confirmed.md
```

Comment:
Is there a working version of this file?

Reply:
yes, it is a PR against `reference` repo - https://github.com/openconfig/reference/pull/198